cmd/sourced/cmd/root.go

```diff
@@ -35,5 +46,11 @@ type Command struct {
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
-	rootCmd.RunMain()
+	if err := rootCmd.Run(os.Args); err != nil {
+		if errors.Cause(err).Error() == workdir.ErrWrongWorkdir.Error() {
```

Comment:
This would be much cleaner if we'd use `src-d/go-errors`, e.g 
```suggestion
		if ErrWrongWorkdir.Is(err) {
```
https://github.com/src-d/go-errors/blob/master/example_test.go#L54

Reply:
I'm ok using `go-errors`